### PR TITLE
Docs: fix typo in important message

### PR DESCRIPTION
**Pull request by Jucesr**
Fix grammar in "Important message"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 > A simple package to map your routes for your expressjs application
 
 ---
-**IMPORTANT: v1.0.2 fixed a security vulnerability. Every version up to v1.0.1 is not save for production. Update your current version to v1.0.2. You can find more information [here](https://github.com/aichbauer/express-routes-mapper/issues/15).**
+**IMPORTANT: v1.0.2 fixed a security vulnerability. Every version up to v1.0.1 is not safe for production. Update your current version to v1.0.2. You can find more information [here](https://github.com/aichbauer/express-routes-mapper/issues/15).**
```
Comment (aichbauer) [inline README.md]: Could you also change 

> v1.0.2

 to 

> v1.0.2 or higher

?

**Review by aichbauer — CHANGES_REQUESTED** [review]
Hey, thank you for your contribution 👍

Would you mind just changing one little thing?

File: README.md (hunk 1 above)
Comment (Jucesr) [reply README.md]: Sure!

**Review by aichbauer — APPROVED** [review]
Thank you 😁

**Pull request merged**